recipes/lume-epics/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "lume-epics" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e07af16f70731d22e81260e34ad4bf2d9d5bfa17503bdb0328bde8745002a31e
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - render-from-template = lume_epics.commands.render_from_template:serve_from_template
+    - serve-from-template = lume_epics.commands.serve_from_template:serve_from_template
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
+    - epics-base
+    - pyepics
+    - pcaspy
+    - p4p
+    - numpy
+    - bokeh
+    - click
+    - lume-model >=0.8
+    - nose >=1.1.2
```

Comment:
Nose as a run dep is odd. Can you check it later and, if that is not needed but specified, open  an issue upstream. (Nose is deprecated BTW.)

Reply:
@ocepaf, we need it because of p4p. I started to address that upstream. For now we need to keep it here! 😞